core/src/main/java/org/axonframework/commandhandling/model/GenericJpaRepository.java

```diff
@@ -268,7 +302,7 @@ public GenericJpaRepository(EntityManagerProvider entityManagerProvider, Aggrega
      */
     public GenericJpaRepository(EntityManagerProvider entityManagerProvider, Class<T> aggregateType, EventBus eventBus,
                                 LockFactory lockFactory, Function<String, ?> identifierConverter) {
-        this(entityManagerProvider, aggregateType, eventBus, null, lockFactory, identifierConverter);
+        this(entityManagerProvider, aggregateType, eventBus, null, null, lockFactory, identifierConverter);
```

Comment:
It would bring a lot of dependencies here since `SimpleDeadlineManager` depends on `DeadlineTargetLoader` which dependes on `RepositoryProvider` and `SagaRepositoryProvider`. That's why I added a new constructor which is used by the configuration.

Reply:
So the simplest answer would be 'no, it doesn't Steven, since...'. Cool, thanks for pointing that out!